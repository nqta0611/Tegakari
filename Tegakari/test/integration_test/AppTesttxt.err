java.util.NoSuchElementException: No line found
	at java.util.Scanner.nextLine(Scanner.java:1585)
	at guiConsoleController.ConsoleLobbyPhase.update(ConsoleLobbyPhase.java:137)
	at java.util.Observable.notifyObservers(Observable.java:159)
	at java.util.Observable.notifyObservers(Observable.java:115)
	at tegakari.Lobby.updatePlayers(Lobby.java:343)
	at tegakari.GameClient.handleMessageFromServer(GameClient.java:279)
	at com.lloseng.ocsf.client.AbstractClient.run(AbstractClient.java:281)
	at java.lang.Thread.run(Thread.java:745)
java.io.EOFException
	at java.io.ObjectInputStream$BlockDataInputStream.peekByte(ObjectInputStream.java:2598)
	at java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1318)
	at java.io.ObjectInputStream.readObject(ObjectInputStream.java:370)
	at com.lloseng.ocsf.client.AbstractClient.run(AbstractClient.java:276)
	at java.lang.Thread.run(Thread.java:745)
