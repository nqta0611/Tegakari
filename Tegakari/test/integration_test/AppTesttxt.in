1
file just for testing. LEAVE THE 1.
