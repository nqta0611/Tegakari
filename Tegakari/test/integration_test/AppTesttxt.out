Client connected
Player joined game
Insert a username:
Lobby name changed to: Tegakari: 1
Player accepted
Please Choose a theme: Greek(G), Washington(W), or Pirate(P)
Would you like to add a robot? (y/n)
Connection exception: No line found, class type: class java.util.NoSuchElementException
file not found, check your arguments for input
file not found, check your arguments for output
file not found, check your arguments for error
New Theme Selected: GREEK
New Theme Selected: GREEK
Include a valid IP address of the target server immediately following the '-server' flag. E.g: -server 129.65.150.243
File Not found pass.
Connection exception: null, class type: class java.io.EOFException
Client disconnected
Client was disconnected
